.github/workflows/build.yml

```
@@ -55,4 +55,4 @@ jobs:
           tags: ${{ steps.meta.outputs.tags }}
           labels: ${{ steps.meta.outputs.labels }}
           cache-from: type=gha
-          cache-to: type=gha,mode=max
+          cache-to: type=gha,mode=max
```

yamllint

[error] no new line character at the end of file

(new-line-at-end-of-file)